linux_os/guide/services/ldap/389_ds/package_389-ds-base_removed/rule.yml

```diff
@@ -0,0 +1,52 @@
+documentation_complete: true
+  
+prodtype: rhel6,rhel7,rhel8
+
+title: 'Uninstall 389-ds-base Package'
+
+description: |-
+    The <tt>389-ds-base</tt> package should be removed if not in use.
+    Is this system the 389-ds server? If not, remove the package.
+    {{{ describe_package_remove(package="389-ds-base") }}}
+    The 389-ds-base RPM is not installed by default on a {{{ full_name }}}
+    system. It is needed only by the 389-ds server, not by the
+    clients which use LDAP for authentication. If the system is not
+    intended for use as an LDAP Server it should be removed.
+
+rationale: |-
+    Unnecessary packages should not be installed to decrease the attack
+    surface of the system.  While this software is clearly essential on an LDAP
+    server, it is not necessary on typical desktop or workstation systems.
+
+severity: low
+
+identifiers:
+    cce@rhel6: 82722-0
+    cce@rhel7: 82723-8
+    cce@rhel8: 82724-6
```

Comment:
CCEs! Nice! Please also add one for ``ocp4``, which will also cover CoreOS.

Reply:
Can do.